### ISSUE TITLE
What does the spine have to do with confreq-rs-foreign_image?

### ISSUE THREAD

**Issue by dlazin** [w3c/epub-tests#72]
The description for https://w3c.github.io/epub-tests/#confreq-rs-foreign_image is:

> An HTML content file contains a PSD image, with a manifest fallback to a PNG image. This tests fallbacks for resources that are not in the spine.

The first half makes sense as a test for https://w3c.github.io/epub-specs/epub33/rs/#confreq-rs-foreign, but I don't understand how the spine is involved. Is the spine part irrelevant, or, if not, should this perhaps be two separate tests (image fallback and resource-not-in-spine fallbacks)?

See https://github.com/w3c/epub-tests/blob/main/tests/confreq-rs-foreign_image/OPS/package.opf for a refresher of what this test is actually doing :)

**Comment by dauwhe** [comment]
The fallback mechanism works in two situations: one is for putting non-CMT resources in the spine, and then falling back to those. The other is providing fallbacks for non-CMT resources that are embedded in regular EPUB content documents. This test is for the latter situation. I expect the code paths for these in reading systems would be really different, if in fact any reading system did this.

**Comment by iherman** [comment]
@dlazin is it o.k. to close this issue?

**Comment by dlazin** [comment]
Yes, it's OK to close; I will do so. Ideally we would change the test to clarify for future readers, but this bug holds the record for now.
